src/components/Core/AppCore/AppCore.js

```diff
@@ -18,15 +20,51 @@ import ConnectivityWrap from '../../Common/Navigation/Connectivity/ConnectivityW
 import ContentSelectorWrap from '../../Common/Misc/ContentSelectorWrap'
 import ContentSelector from '../../Common/Misc/ContentSelector'
 
+import AuthLoadingOverlay from '../../Common/Misc/AuthLoadingOverlay'
+import AuthRedirectPopup from '../../Common/Misc/AuthRedirectPopup'
+import AuthLoggedOutPopup from '../../Common/Misc/AuthLoggedOutPopup'
+import { useAuth0 } from '@auth0/auth0-react'
+
 export default function AppCore({ agent }) {
   const [configuredOrigin, setConfiguredOrigin] = useState('')
   const [data, setData] = useState({})
   const [status, error, startFetchHandler] = useGetServerStatus()
 
+  const [showAuthRedirectPopup, setShowAuthRedirectPopup] = useState(false)
+  const [showAuthLoggedOutPopup, setShowAuthLoggedOutPopup] = useState(false)
+
+  const { loginWithRedirect, isAuthenticated, isLoading, logout, user } =
+    useAuth0()
+
   /* Example of data obj: */
-  /* {"version":"0.7.1","label":"faber.agent", "conductor":{"in_sessions":0,"out_encode":0,
+  /* {"version":"0.7.3","label":"foo.agent", "conductor":{"in_sessions":0,"out_encode":0,
 	"out_deliver":0,"task_active":1,"task_done":816, "task_failed":97,"task_pending":0}} */
 
+  const clickLogoutHandler = () => {
+    logout()
```

Comment:
I think this needs a 'returnTo' option to stay on the same port. Right now it defaults to `localhost:3000` as it's first in the list of 'Allowed Logout URLs', which is fine, but in the demo the front-end will be running on a different port

Reply:
Fixed